frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -339,211 +342,212 @@ export function PollWorkerScreen({
     );
   }
 
-  const pollworkerMainScreen = (
```

Comment:
this diff is mostly undoing a change I had made when I added the diagnostics screen. previously, I had only rendered the sidebar in PollWorkerScreen and then had the main panel render either the poll worker main screen or the diagnostics screen. That didn't work anymore with the full-screen accessible controller diagnostic, so now the diagnostics screen has to render the sidebar itself. this could probably be handled better with an app-wide router, but i think this works for what we've got